spec/features/create_weather_spec.rb

```diff
@@ -1,30 +1,34 @@
 require 'rails_helper'
+require 'vcr'
 
-feature 'User adds a new weather', %Q{
-  As an authenticated user
-  I want to add a weather
-  So that others can review it
-  } do
+  feature 'User adds a new weather', %Q{
+    As an authenticated user
+    I want to add a weather
+    So that others can review it
+    } do
 
-  before :each do
-    @user = FactoryGirl.create(:user)
-    login_as(@user)
-  end
+    before :each do
+      @user = FactoryGirl.create(:user)
+      login_as(@user)
+    end
+
+    scenario 'with valid data' do
+      visit '/weathers/new'
+      weather = FactoryGirl.create(:weather)
+
+      visit "/weathers"
+
+      expect(page).to have_content weather.city
+      expect(page).to have_content weather.date.strftime("%B %d %Y")
+      expect(page).to have_css ("span img")
+    end
 
-  scenario 'with valid data' do
-    weather = FactoryGirl.create(:weather)
-    visit '/weathers/new'
-    fill_in 'City', with: weather.city
-    fill_in 'State', with: weather.state
-    select '2014', from: 'weather_date_1i'
-    select 'December', from: 'weather_date_2i'
-    select '29', from: 'weather_date_3i'
-    fill_in 'Conditions', with: weather.conditions
-    fill_in 'Feel', with: weather.feel
-    click_button 'Create New Weather'
+    scenario 'with invalid city/state' do
+        visit '/weathers/new'
+        fill_in "City", with: "Boston"
+        select 'California', from: "State"
+        click_button 'Create New Weather'
 
-    expect(page).to have_content 'New weather successfully submitted'
-    expect(page).to have_content weather.city
-    expect(page).to have_content weather.date.strftime("%B %d %Y")
+        expect(page).to have_content "Something went wrong with your request. Please check city and state."
+      end
```

Comment:
`end` at 33, 6 is not aligned with `scenario 'with invalid city/state' do` at 26, 4